orbit/pkg/execuser/execuser_linux.go

```diff
@@ -24,10 +24,13 @@ func run(path string, opts eopts) error {
 		Int64("id", user.id).
 		Msg("running sudo")
 
-	arg := []string{"-u", user.name, "-H"}
+	// Flag `-i` is needed to run the command with the user's context, from `man sudo`:
+	// "The command is run with an environment similar to the one a user would receive at log in"
+	arg := []string{"-i", "-u", user.name, "-H"}
```

Comment:
could we use `--login` for extreme clarity?

```suggestion
	arg := []string{"--login", "-u", user.name, "-H"}
```